### ISSUE TITLE
Invalid Form Body content

### ISSUE THREAD

**Issue by navarr** [Shotbow/ChatBot#56]
```
events.js:171
    throw err; // Unhandled 'error' event
    ^

Error [ERR_UNHANDLED_ERROR]: Unhandled error. ([object Object])
    at Client.emit (events.js:169:17)
    at WebSocketConnection.onError (/usr/lib/chatbot/node_modules/discord.js/src/client/websocket/WebSocketConnection.js:374:17)
    at WebSocket.onError (/usr/lib/chatbot/node_modules/ws/lib/event-target.js:128:16)
    at WebSocket.emit (events.js:180:13)
    at _receiver.cleanup (/usr/lib/chatbot/node_modules/ws/lib/websocket.js:211:14)
    at Receiver.cleanup (/usr/lib/chatbot/node_modules/ws/lib/receiver.js:557:13)
    at WebSocket.finalize (/usr/lib/chatbot/node_modules/ws/lib/websocket.js:206:20)
    at TLSSocket.emit (events.js:180:13)
    at emitErrorNT (internal/streams/destroy.js:64:8)
    at process._tickCallback (internal/process/next_tick.js:178:19)
Successfully logged in!
(node:1581) UnhandledPromiseRejectionWarning: DiscordAPIError: Invalid Form Body
content: Must be 2000 or fewer in length.
    at item.request.gen.end (/usr/lib/chatbot/node_modules/discord.js/src/client/rest/RequestHandlers/Sequential.js:71:65)
    at then (/usr/lib/chatbot/node_modules/snekfetch/src/index.js:215:21)
    at <anonymous>
    at process._tickCallback (internal/process/next_tick.js:182:7)
(node:1581) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). (rejection id: 3)
(node:1581) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
(node:1581) UnhandledPromiseRejectionWarning: DiscordAPIError: Invalid Form Body
content: Must be 2000 or fewer in length.
    at item.request.gen.end (/usr/lib/chatbot/node_modules/discord.js/src/client/rest/RequestHandlers/Sequential.js:71:65)
    at then (/usr/lib/chatbot/node_modules/snekfetch/src/index.js:215:21)
    at <anonymous>
    at process._tickCallback (internal/process/next_tick.js:182:7)
(node:1581) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). (rejection id: 5)
```

**Comment by navarr** [comment]
I think this might be related to whowas lookups where there are far too many usernames

**Comment by McJeffr** [comment]
Opened a PR with a possible fix: #57
